gm4_speed_paths/data/gm4_speed_paths/functions/init.mcfunction

```diff
@@ -3,9 +3,6 @@ execute unless score speed_paths gm4_earliest_version < speed_paths gm4_modules
 scoreboard players set speed_paths gm4_modules 1
 
 scoreboard objectives add gm4_speed_paths dummy
-
-schedule function gm4_speed_paths:main 1t
-
-
+scoreboard objectives add gm4_speed_paths_leave minecraft.custom:leave_game
```

Comment:
What is this scoreboard used for? Is it reliable? A client forcefully disconnecting for example will not cause this scoreboard to increase.

Reply:
When a player logs out with the path score, which keeps the clock active, there are no remaining players having the score and the clock stops.  But since they still keep the score, if the clock is not running when they rejoin, they would need to wait for someone else.

Using the rejoin, I clear their existing score so they can start up the clock immediately if they logged out off a path